mozilla/ichnaea
# Remove mysqlconnector requirement and usage

mysqlconnector was added a couple of years ago for the datamap and dump
scripts. However, it adds a lot of complexity to the codebase and forces
us to keep tabs on two different mysql libraries.

A lot of things have changed in the last couple of years in the world of
Python and mysql clients and we think the compelling reasons to have two
are gone now. Thus we're going to simplify the situation and switch back
to PyMySQL for all mysql connections.

Fixes #870

To test:

1. `make build` and `make setup`
2. `make lint` and `make test`
3. `make run` and run through the site--though it's all static pages for the most part
4. `make runcelery` and make sure the scheduler and workers do things

This doesn't test the datamap and dump scripts outside of whatever gets tested in the test suite. We'll test datamap more thoroughly as part of #503 and #840. If there are issues, we can figure them out there.
